Makefile

```diff
@@ -22,6 +21,9 @@ $(MAKEFILE):
 	git clone --quiet --depth 1 -b $(CI_BRANCH) $(CI_REPOSITORY) $(CI_PATH);
 -include $(MAKEFILE)
 
+dependencies:
+	$(MAKE) -C tools dependencies
```

Comment:
Can you explain to me what this line does? I'm not very familiar with makefiles yet.

Reply:
Oh ok, I guess that it is equivalent to `cd tools && make dependencies` right?

Reply:
you understood it correctly.